test/Regex/Bug15992535.baseline

```diff
@@ -0,0 +1 @@
+Expecting fail fast

```

Comment:
side note: long-term we should probably move this to throwing a catchable error instead, but this matches previous behavior.

Reply:
Yes, I agree; I've created #4722 for this follow-up work.